configuration/display_rotation.md

```diff
@@ -0,0 +1,34 @@
+## How to rotate your display
+
+The options to rotate the display of your Raspberry Pi depend on which display driver software it is running, which may also depend on which Raspberry Pi you are using. 
+
+### Fake or Full KMS graphics driver (Default on Pi4)
+
+If you are running the Raspberry Pi desktop then rotation is achieved by using the `Screen Configuration Utility` from the desktop `Preferences` menu. This will bring up a graphical representation of the display or displays connected to the Raspberry Pi. Right click on the display you wish to rotate and select the required option.
+
+It is also possible to change these settings using the command line `xrandr` option. The following commands give 0°, -90°, +90° and 180° rotations respectively. 
```

Comment:
Does the "0°, -90°, +90° and 180° rotations" list match the order of the commands below? If not, maybe the commands below should be re-ordered to match.

Reply:
Doh!